core/device_hooks/cuda_hooks.cpp

```diff
@@ -110,8 +110,10 @@ void CudaExecutor::synchronize() const GKO_NOT_COMPILED(cuda);
 
 void CudaExecutor::run(const Operation &op) const
 {
+    this->template log<log::Logger::operation_launched>(this, &op);
```

Comment:
These changes were left-over from cherry-picking the named lambda commit from #759, they are not really necessary here.